rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/match_changed_variable.php.inc

```diff
@@ -23,8 +28,13 @@ namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromo
 
 class MatchChangedVariable
 {
-    public function __construct(public float $x = 0.0)
+    public function __construct(public float $a = 0.0)
```

Comment:
this should not be changed, `public` property is accessible from outside object consumer, so should be kept as `public float $x = 0.0`